_data/accessibility-tests/side-navigation.yml

```diff
@@ -0,0 +1,66 @@
+title: Side navigation
+component_status: pass
+test_items:
+  # General tests
+  - summary: Side navigation appears in the same location and order on every page of the website.
+    summary_additional: When you navigate to different pages on the website, the side navigation is always in the same order and in the same place.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 3.2.3
+  - summary: Side navigation clearly indicates location on the website.
+    summary_additional: When viewing side navigation content, the links have distinct styling that give clear visual indication of where you are in the site. There are also other visual elements to further indicate where you are in relation to the navigation tree.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.8
+  - summary: Side navigation link text and background meet color contrast requirements.
+    summary_additional: When you use a color contrast checker to view the side navigation, contrast between the text and background colors is at least 4.5:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.3
+  - summary: Color is not the only method used to convey side navigation link states.
+    summary_additional: When viewing the side navigation links, you can read text to understand what page you are on. There may also be underlining missing to indicate you are on the current page.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.1
+  # Keyboard navigation tests
+  - summary: Keyboard can access all links in the side navigation.
+    summary_additional: When you navigate through the side navigation with a keyboard, you can interact with every link inside it.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.1.1
+  - summary: Side navigation does not trap keyboard focus.
+    summary_additional: When you navigate through side navigation links with a keyboard, you can easily move into and out to get to other content.
+    test_status: conditional
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.1.2
+  - summary: Focus indicator is clearly visible in side navigation.
+    summary_additional: When you use a keyboard to tab into the side navigation, the links have a visible underline, bold style or other clear indication that links are interactive.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.7
```

Comment:
**question (non-blocking):** Was there a reason we changed the Gherkin for focus indicators? This mentions a visible underline and bold style but neither is the default for focus indicators. Most of our other descriptions for `2.4.7` read:

>**Focus indicator is clearly visible.** When you use a keyboard to navigate through the header, there will be a visible outline or other clear indication where the the focus is. 

 @amycole501  @alex-hull @finekatie

Reply:
Hmm...I'm not sure why that one got changed. I think it should reflect the way others have been written. The previous way potentially helps with some generalizing, versus tying people to those options.

Reply:
Let me know if this change is what you had in mind! [77b383e](https://github.com/uswds/uswds-site/pull/2864/commits/77b383ec6991c2c440096a1d3188ad9887b846b6)